### ISSUE TITLE
.NET 8: Code only EF with RP add inheritance and advanced topics

### ISSUE THREAD

**Issue by Rick-Anderson** [dotnet/AspNetCore.Docs#26505]
1.  Copy Code from [EF-RP 6.0  project](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/data/ef-rp/intro/samples/cu60)  to [cu90](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/data/ef-rp/intro/samples/cu90) and update for .NET 9 

2. Quick review of code and update where necessary. 
3. Add code for inheritance. Hopefully you can use some of the code from [Tutorial: Implement inheritance - ASP.NET MVC with EF Core](https://docs.microsoft.com/en-us/aspnet/core/data/ef-mvc/inheritance).
4. Consider adding code supporting [Tutorial: Learn about advanced scenarios - ASP.NET MVC with EF Core](https://docs.microsoft.com/en-us/aspnet/core/data/ef-mvc/advanced?view=aspnetcore-6.0).

Once the preceding is complete, ~we'll port all the RP code and update the [Tutorial: Get started with EF Core in an ASP.NET MVC web app](https://docs.microsoft.com/en-us/aspnet/core/data/ef-mvc/intro?view=aspnetcore-6.0)~
and make a Blazor version.

@guardrex - @tdykstra 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7a54c97f-ddd4-b041-5e5f-a2df009ace50
* Version Independent ID: fa488196-1f98-08ea-adcc-17d9dac86727 and 3e28328b-9c7a-359b-2f57-319b0bc7eea3
* Content: [Razor Pages with Entity Framework Core in ASP.NET Core - Tutorial 1 of 8](https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/intro) and https://learn.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/?view=aspnetcore-8.0
* Content Source: [aspnetcore/data/ef-rp/intro.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/data/ef-rp/intro.md) and https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/razor-pages/index.md
* Product: **aspnet-core**
* Technology: **aspnetcore-data**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

---

#### Document Details

---
[Associated WorkItem - 33228](https://dev.azure.com/msft-skilling/Content/_workitems/edit/33228)

**Comment by Rick-Anderson** [comment]
@JeremyLikness  I'm hoping you can do this early next year after the .NET 7 push. Would it be helpful to create an issue tacking this in https://github.com/dotnet/EntityFramework.Docs/issues ?
cc @roji @ajcvickers

**Comment by ajcvickers** [comment]
@Rick-Anderson What is the value of tracking it in the EF repo? (Happy to have one if there is a reason for it.)

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson What is the value of tracking it in the EF repo? (Happy to have one if there is a reason for it.)

I was thinking you folks don't check other repo's for issues. I'm fine with tracking it here only. I'm mention you next year.

**Comment by Rick-Anderson** [comment]
@JeremyLikness are you the right person to update the RP/EF code?

**Comment by Rick-Anderson** [comment]
@ajcvickers who should I assign this task, code writing only since @JeremyLikness  is now in a new role? We're happy to do two hours of editing and scut work in https://github.com/dotnet/EntityFramework.Docs for every hour the EF team spends coding.
cc @tdykstra

**Comment by ajcvickers** [comment]
@Rick-Anderson It will have to be someone on the dev team. We'll prioritize it alongside other docs work.

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson It will have to be someone on the dev team. We'll prioritize it alongside other docs work.

@ajcvickers  mention me and @tdykstra  on doc issues in https://github.com/dotnet/EntityFramework.Docs and we'd be happy to help to free up time for the EF Core code update in this topic.

**Comment by LayersOfAbstraction** [comment]
Hi guys I hope I found the right place to create new issues to get feedback on. I have a bunch of ideas that I will separate into each issue. I will try and do it here.

In regards to my attempt to migrate the MVC doco from .NET 5 to .NET 6, I see a couple of options.

We could create the whole ContosoUniversity sample solution app from scratch which would make developing it probably easier and call it say '6cu'.

Or we could slowly migrate the application cu at aspnetcore\data\ef-mvc\intro\samples\cu from .NET 2.2 to .NET 6 but it may mean wiping out some previous work. I think the first approach may be the better approach just so the previous contributions are preserved. What are your thoughts, on upgrading CU to .NET6, please?

**Comment by Rick-Anderson** [comment]
@ajcvickers 
> @Rick-Anderson It will have to be someone on the dev team. We'll prioritize it alongside other docs work.

Happy to quid pro quo 2:1 on your effort here. @tdykstra  and I will spend 2 hours on https://github.com/dotnet/EntityFramework.Docs/issues for every hour it takes your team.

**Comment by Rick-Anderson** [comment]
Cleaning up with new issue #4633

**Comment by cremor** [comment]
@Rick-Anderson Could you please fix the issue number in your last comment? The one which is written now is a PR from 2017.
